README.rst

```diff
@@ -1407,6 +1407,8 @@ to any key of a file. When set, all values underneath the key that set the
 Note that, while in cleartext, unencrypted content is still added to the
 checksum of the file, and thus cannot be modified outside of sops without
 breaking the file integrity check.
+This behavior can be modified using `--mac-only-encrypted` flag which makes
+sops compute a MAC only over values it encrypted and not all values.
```

Comment:
It can also be controlled by the `.sops.yaml` config, which should also be mentioned.